manifests/c/CrystalIDEASoftware/UninstallTool/3.6.0.5681/CrystalIDEASoftware.UninstallTool.installer.yaml

```diff
@@ -2,12 +2,12 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 
 PackageIdentifier: CrystalIDEASoftware.UninstallTool
-PackageVersion: 3.5.10
+PackageVersion: 3.6.0.5681

```

Comment:
The version written in the registry is 3.6.0, I think it'd be better to have same value here.

Reply:
Done!